packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -254,10 +263,11 @@ async function _getLatestBlockNumber() {
 }
 
 // Add additional environment variables for a given config file. Used to attach starting and ending block numbers.
-function _appendBlockNumberEnvVars(config, lastQueriedBlockNumber, latestBlockNumber) {
+function _appendEnvVars(config, botName, lastQueriedBlockNumber, latestBlockNumber) {
```

Comment:
this function now adds the `BOT_IDENTIFIER` as the key from the config object. this will clean up the configs as we no longer need to provide the `BOT_IDENTIFIER` over and over again as this is a duplicate from the key in the main json object.